### PR TITLE
Fix Issue #568:Retaining state on Orientation changes

### LINKED ISSUE #568: Retaining activity state on orientation changes and preventing crash on orientation changes
**Summary:**

Currently The state is lost on changing the orientation ie from landscape to portrait or vice versa.

**Steps to reproduce:**

For Instance
1)Run the app
2)Third party transfers
3)Select some option(continue)
4)Change orientation
Changes get reset

**Expected behaviour:**

The changes should be retained on orientation changes.

**Observed behaviour:**

The changes are reset on orientation changes. This behaviour is persistent across many activities.

**Device and Android version:**

Android 5.0.2,API 21

**Screenshots:**

I am working on this

### PR DESCRIPTION
**Pull request by AkshGautam**
Fixes #568 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by AkshGautam** [comment]
@dilpreet96 @therajanmaurya  Please review.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="org.mifos.mobilebanking">
+    package="org.mifos.mobilebanking">
```
Comment (luckyman20) [inline app/src/main/AndroidManifest.xml]: Please do not change the spacing

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -72,7 +83,7 @@
         </receiver>
 
         <service android:name="org.mifos.mobilebanking.ui.widgets.ChargeWidgetService"
-                 android:permission="android.permission.BIND_REMOTEVIEWS" />
+            android:permission="android.permission.BIND_REMOTEVIEWS" />
```
Comment (luckyman20) [inline app/src/main/AndroidManifest.xml]: Here also, spacing

**Review by luckyman20 — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (AkshGautam) [reply app/src/main/AndroidManifest.xml]: @luckyman20 I thought Travis checks would find that out but it seems Travis check does not work for XML layouts. any more spaces which I have left?

**Pull request merged**